journal/modernized/21-christs-kingdom-exalted.adoc

```diff
@@ -1,9 +1,12 @@
 == Christ`'s Kingdom Exalted;
 
+[.blurb]
 === Being a Short Essay in Order to Promote Truth and Righteousness in the Earth, and to Discourage Unholiness, Ungodliness, and Unbelief.
```

Comment:
I really wasn't sure what to do with this. It didn't look right and I scrolled through the style guide to see if anything else would work, but this is what I came up with. It seems the Preface is too close to it and I had [.offset] it and it looked better, but I don't remmember seeing a title done that way before so I took it out.